src/ageml/ui.py

```diff
@@ -44,18 +44,42 @@ class Interface:
     ---------------
     setup(self): Creates required directories and files to store results.
 
+    command_setup(self, dir_path): Create required directories and files to store results for command.
+
     set_flags(self): Set flags.
 
     set_visualizer(self): Set visualizer with output directory.
 
+    set_dict(self): Initialise dictionaries for data storage.
+
+    set_features_dataframes(self): Set features dataframes for each subject type, covariate and system.
+
     generate_model(self): Set model with parameters.
 
     set_classifier(self): Set classifier with parameters.
 
+    update_params(self): Update initial parameters after load.
+
     check_file(self, file): Check that file exists.
 
     load_csv(self, file): Use panda to load csv into dataframe.
 
+    load_features(self, required): Load features from csv file.
+
+    load_covariates(self, required): Load covariates from csv file.
+
+    load_clinical(self, required): Load clinical from csv file.
+
+    load_factors(self, required): Load factors from csv file.
+
+    load_ages(self, required): Load ages from csv file.
+
+    system_parser(self, file): Parse systems file.
+
+    load_systems(self, required): Load systems file.
+
+    remove_missing_data(self): Remove subjects with missing values.
```

Comment:
`Interface.remove_missing_data` method will probably get deprecated in future versions that include inputation methods or other missingness handling strategies.